vhdbuilder/packer/test/run-test.sh

```diff
@@ -71,12 +71,26 @@ FULL_PATH=$(realpath $0)
 CDIR=$(dirname $FULL_PATH)
 if [ "$OS_TYPE" == "Windows" ]; then
     SCRIPT_PATH="$CDIR/$WIN_SCRIPT_PATH"
-    ret=$(az vm run-command invoke --command-id RunPowerShellScript \
-        --name $VM_NAME \
-        --resource-group $RESOURCE_GROUP_NAME  \
-        --scripts  @$SCRIPT_PATH \
-        --output json \
-        --parameters "containerRuntime=${CONTAINER_RUNTIME}" "WindowsSKU=${WINDOWS_SKU}")
+    retry=0
+    maxRetry=2
+    while [ $retry -lt $maxRetry ]; do
+        ret=$(az vm run-command invoke --command-id RunPowerShellScript \
+            --name $VM_NAME \
+            --resource-group $RESOURCE_GROUP_NAME  \
+            --scripts  @$SCRIPT_PATH \
+            --output json \
+            --parameters "containerRuntime=${CONTAINER_RUNTIME}" "WindowsSKU=${WINDOWS_SKU}")
```

Comment:
Do you know why the docker is not running? This may be an issue

Reply:
Not sure, my guess is starting docker takes a while, so adding at most two retries here.

Reply:
We can print any debuggable info after a failure to check the docker daemon's status to see if it will come up or never.

Reply:
Could we add code in `vhd-content-test.ps1` to wait the docker running before calling `docker images`? Retry here may not be efficient

Reply:
Added retry logic inside `vhd-content-test.ps1`